### DISCUSSION TITLE
Wyze app for remote access

### DISCUSSION THREAD

**Discussion by Neophema** [gtxaspec/wz_mini_hacks#311] [Q&A]
Not sure where I am going wrong but the IOS app fails. 

My V3 & V2Pan works well with wz_mini_hacks RTSP and I can see it on iSpy. 

The V3 always works with wz_mini_hacks and the IOS app. The V2Pan stops authenticating to the Wyze IOS app after a few hours but the RTSP keeps working. This tells me the camera is fine but the Wyze protocol is failing.  Is this the hack affecting the Wyze software? 

The IOS app is the only remote access I have to the cameras so I lose access to the V2Pan. What am I doing incorrectly?

**Comment by maxfield-allison** [comment]
FWIW,
I had about 4 v2's that would never do rtsp with the hack to anything but vlc. my v2 pan cams work flawlessly as do all but one of my v3's. the difference seems to be the firmware version. All this to say, we're still kind of in beta with this project and the variables introduced by different hardware and firmware are going to cause some issues like this. Therefor, I think your best (only?) course of action may be downgrading firmware and contributing your findings/logs. 

As a workaround to wyze being your only remote viewing option, you could also set up a reverse proxy with oauth and a dynamic dns service or you're own cheap $10/year domain name to access your nvr software, or configure a vpn on your router/firewall for your phone/pc. I'm currently using frigate (nvr with ai object detection) and traefik (reverse proxy for all my docker services, includes acme and letsencrypt) via cloudflare with google oauth all on docker. If you're interested I can share more info on getting it set up.

**Reply by Neophema** [reply]
Yes please and thank you. 

I am interested in frigate as well so right on point.

**Reply by maxfield-allison** [reply]
https://github.com/htpcBeginner/docker-traefik

This guy is the most thorough and explains basically everything you'd ever need to know to get up and running. I followed his guides back in 2017 or so to get my docker server started. It's evolved a ton since then but it might still be good to go through the guides and articles from the beginning. you can effectively skip anything you already know or apps you don't need and just keep what you do use.

**Reply by Neophema** [reply]
Thank you I will look it over.
